lib/puppetserver/ca/utils/config.rb

```diff
@@ -0,0 +1,13 @@
+module Puppetserver
+  module Ca
+    module Utils
```

Comment:
How do people feel about putting all parent modules on one line to reduce indentation, e.g. 
```
module Puppetserver::Ca::Utils
  module Config
  end
end
```

Reply:
no feelings either way

Reply:
We played with this and combining the names messed with Ruby's constant lookup so we went with the old way.